sys-devel/hcc/hcc-2.7.0.ebuild

```diff
@@ -17,22 +17,27 @@ IUSE="debug"
 CMAKE_BUILD_TYPE=Release
 
 RDEPEND="=dev-libs/rocr-runtime-${PV}*
-	 dev-util/rocminfo"
+	dev-util/rocminfo"
 DEPEND="${RDEPEND}
 	dev-util/cmake
 	dev-vcs/git"
 
+
 src_configure() {
 	strip-flags
 	if ! use debug; then
 		append-cflags "-DNDEBUG"
 		append-cxxflags "-DNDEBUG"
+		buildtype="Debug"
+	else
+		buildtype="Release"
 	fi
 
 	local mycmakeargs=(
 		-DBUILD_SHARED_LIBS=OFF
```

Comment:
Is there a reason why hcc is set to not being a shared lib?

Reply:
There was a prolem with finding the right libraries. This is the way AMD does it.

Reply:
The problem was, that the libraries from HCC collide with system llvm/clang. Maybe it could be solved using a a "LLVM_VERSION_SUFFIX" (see e.g. llvm-roc-2.8.0.ebuild). But currently I do not spend much time on this, because it works and HCC is marked deprecated.

Reply:
ok, thanks.